# robot position information message from DrRobot Robot.
# it could be from encoder dead reckoning, or from Indoor GPS
Header header    	# timestamp in the header is the time the 

#make sure below sensors on your robot or not
float32 robot_x	# robot x position reading
float32 robot_y	#robot y position reading
float32 robot_heading	# robot heading reading

